Cargo.toml

```
@@ -7,17 +7,18 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-casper-node = { git = "https://github.com/casper-network/casper-node", tag = "v1.5.4" }
-casper-execution-engine = { git = "https://github.com/casper-network/casper-node", tag = "v1.5.4" }
-casper-types = { git = "https://github.com/casper-network/casper-node", tag = "v1.5.4" }
-casper-hashing = { git = "https://github.com/casper-network/casper-node", tag = "v1.5.4" }
+casper-node = { git = "https://github.com/casper-network/casper-node", branch = "feat-2.0" }
+casper-execution-engine = { git = "https://github.com/casper-network/casper-node", branch = "feat-2.0" }
+casper-types = { git = "https://github.com/casper-network/casper-node", branch = "feat-2.0" }
```

```suggestion
casper-node = { git = "https://github.com/casper-network/casper-node", branch = "release-2.0.0-rc6" }
casper-execution-engine = { git = "https://github.com/casper-network/casper-node", branch = "release-2.0.0-rc6" }
casper-types = { git = "https://github.com/casper-network/casper-node", branch = "release-2.0.0-rc6" }
```

There will be more changes into rc6 soon, so we'll bump the deps once again